tools/summaries/README.md

```diff
@@ -0,0 +1,9 @@
+# Generate Place Summaries
+
+This script generates NL summaries for places.
+
```

Comment:
Should also specify here the environment values that the scripts rely on (and assert if it's unavailable).

Reply:
Good idea. Done.